src/components/GoalCreateForm/GoalCreateForm.tsx

```diff
@@ -196,6 +196,7 @@ const GoalCreateForm: React.FC<GoalCreateFormProps> = ({ title, onGoalCreate, pr
                             <DropdownPanel
                                 placement="top-end"
                                 items={createOptions}
+                                mode="single"
```

Comment:
If `mode` prop use `'single'` value too many times then may make as default value for `<DropdownPanel />` component